src/BinaryParsers/ElfBinary/ElfBinary.cs

```diff
@@ -58,12 +58,13 @@ public ElfBinary(Uri uri, string localSymbolDirectories = null, bool forceCompre
 
                 CompilationUnits = new Lazy<List<DwarfCompilationUnit>>(()
                     => LoadDebug(DwarfSymbolProvider.ParseAllCompilationUnits(this,
-                                                                    DebugData,
-                                                                    DebugDataDescription,
-                                                                    DebugDataStrings,
-                                                                    DebugLineStrings,
-                                                                    DebugStringOffsets,
-                                                                    NormalizeAddress), localSymbolDirectories));
+                                                                              DebugData,
+                                                                              DebugDataDescription,
+                                                                              DebugDataStrings,
+                                                                              DebugLineStrings,
+                                                                              DebugStringOffsets,
+                                                                              NormalizeAddress),
+                                                                              localSymbolDirectories));
```

Comment:
this is just a format fix based on the comment I got.

Reply:
should the parameter `localSymbolDirectories` align with first parameter `DwarfSymbolProvider.ParseAllCompilationUnits(....)`?